audio/src/main/res/layout/view_number_pad_numeric_element.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/number_frame"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/marginStandard"
+    tools:context=".AudioActivity"
+    tools:layout_gravity="center"
+    >
+
+    <TextView
+        android:id="@+id/number_content"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_margin="1dp"
+        android:background="@drawable/number_pad_circle"
+        android:fontFamily="@font/ideal_sans_medium"
+        android:gravity="center"
+        android:includeFontPadding="false"
+        android:maxLength="2"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        android:textSize="26sp"
```

Comment:
Same as above. Extract the value to dimens.

Reply:
same as above on my comment tooo.